client/resources/at-commands.mjs

```diff
@@ -92,7 +92,7 @@ export class commandsAPI {
               command = keys[command];
               if (typeof command === 'undefined') {
                 throw new Error(
-                  `Key instruction identifier "${c}" for AT "${assistiveTech.name}", mode "${mode}", task "${task}" is not an available identifier. Update your commands.json file to the correct identifier or add your identifier to resources/keys.mjs.`
+                  `Key instruction identifier "${c}" for AT "${assistiveTech.name}", mode "${mode}", task "${task}" is not an available identifier. Update your commandsV1.json file to the correct identifier or add your identifier to resources/keys.mjs.`
```

Comment:
Well the [`tests/commands.json`](https://github.com/w3c/aria-at/blob/master/tests/commands.json) is for controlling the defining keys used in v2 while [`tests/resources/keys.mjs`](https://github.com/w3c/aria-at/blob/master/tests/resources/keys.mjs) is where the v1 key commands are defined (and then translated later on to some json).

So unless we're renaming the `commands.json` to `commandsV2.json` to make things clearer, this message can stay as is.

If that does get updated though, I think it would be preferred to have a context aware error message depending on the version, to avoid that general statement.